examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL/SDL.h>
```

Comment:
> you can simply re-run it again

I see it doesn't help for you. The error is about CMake was not able to handle `https`. You can use instructions similar to Hunter testing to download appropriate version:
* https://github.com/ruslo/hunter/blob/0380dbd81b80007f4c9c3d01e309d9820a833619/.travis.yml#L74